generator/routing_index_generator.cpp

```diff
@@ -263,8 +267,58 @@ void CalcCrossMwmTransitions(string const & mwmFile, string const & mappingFile,
                              vector<CrossMwmConnectorSerializer::Transition<connector::TransitId>> & transitions)
 {
   CHECK(mappingFile.empty(), ());
-  NOTIMPLEMENTED();
-  // @todo(bykoianko) Filling |transitions| based on transit section should be implemented.
+  try
+  {
+    FilesContainerR cont(mwmFile);
+    CHECK(cont.IsExist(TRANSIT_FILE_TAG),
+          (TRANSIT_FILE_TAG, "should be generated before", TRANSIT_CROSS_MWM_FILE_TAG));
+    auto reader = cont.GetReader(TRANSIT_FILE_TAG);
+    ReaderSource<FilesContainerR::TReader> src(reader);
+
+    transit::FixedSizeDeserializer<ReaderSource<FilesContainerR::TReader>> numberDeserializer(src);
```

Comment:
Plz, ```c++ using Source = ReaderSource<...>``` because of duplication. Also, I suggest you to do following:
```c++
transit::TransitHeader header
{
  transit::FixedSizeDeserializer<Source> des(src);
  des(header);
}
```
Same for deserialization of stops and edges below.

Reply:
> `using Source = ReaderSource<...>`

 Done

> I suggest you to do following: `transit::TransitHeader header { transit::FixedSizeDeserializer<Source> des(src); ...` 

Done

> Same for deserialization of stops and edges below.

Done